core/templates/pages/topic-editor-page/editor-tab/topic-editor-stories-list.directive.html

```diff
@@ -30,7 +30,9 @@
       </a>
     </td>
     <td>
-      <span ng-click="deleteCanonicalStory(summary.getId())" class="fas fa-trash-alt list-summary"> </span>
+      <a ng-click="deleteCanonicalStory(summary.getId())" class="list-summary">
+        <span class="fas fa-trash-alt delete-icon"> </span>
```

Comment:
Do we still need the `delete-icon` class here?

Reply:
You are right, we don't need it, I have remove it. If in the future we would need to associate more attributes to the icon, we can add the class later.
Thanks!